spring-boot/admin/src/main/resources/templates/admin/fragments/consumer_wizard/wizard.html

```diff
@@ -115,7 +115,7 @@ <h2>
                                             </tr>
                                             </thead>
                                             <tbody>
-                                            <tr>
+                                            <tr th:if="${testService != null}">>
```

Comment:
It makes sense to hide this table row in case testService is null. In addition, it would be good to display an error message like "Currently the tutorial test service is not correctly configured", so that the problem will be clear to website visitors (and developers).

Reply:
fixed